docs/zero_to_hero_guide/README.md

```diff
@@ -73,7 +73,7 @@ If you're looking for more specific topics, we have a [Zero to Hero Guide](#next
    Open a new terminal and install `llama-stack`:
    ```bash
    conda activate ollama
-   pip install llama-stack==0.1.0
+   pip install llama-stack==0.1.3
```

Comment:
oh good spot. let's get rid of this version please. no need to pin it. just say `pip install -U llama-stack`

Reply:
Thank you for the suggestion @ashwinb . I have made the suggested  change. Ptal.